### ISSUE TITLE
The "privacy" plugin is not installed for mkdocs-material-9.5.0

### ISSUE THREAD

**Issue by shenweiyan** [squidfunk/mkdocs-material#6464]
### Bug description

mkdocs-material-9.5.0 by default added the privacy plugin, but when I add the following lines to `mkdocs.yml`:
```
plugins:
  - privacy
```
`mkdocs serve` will prompt the following error message:
```
RROR   -  Config value 'plugins': The "privacy" plugin is not installed

Aborted with 1 configuration errors!
```

### Related links

- https://squidfunk.github.io/mkdocs-material/plugins/privacy/
- https://github.com/squidfunk/mkdocs-material/releases/tag/9.5.0

### Reproduction

[9.5.0-privacy-errors-mkdocs-material.zip](https://github.com/squidfunk/mkdocs-material/files/13608017/9.5.0-privacy-errors-mkdocs-material.zip)

### Steps to reproduce

`mkdocs serve`

### Browser

Chrome

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
Thanks for reporting. I forgot something during the merge, so this was a rookie mistake on my behalf. Fixed in ac41e53b4.

**Comment by squidfunk** [comment]
Released as part of [9.5.1](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.1).
